vscode/src/chat/MessageProvider.ts

```diff
@@ -783,6 +782,11 @@ export abstract class MessageProvider extends MessageHandler implements vscode.D
 
         return DEFAULT_MAX_TOKENS - solutionLimit
     }
+
+    public logTelemetryService(eventName: string, properties?: any, hasVSEvent = true): void {
```

Comment:
```suggestion
    public logTelemetryService(eventName: string, properties?: Record<string, unknown>, hasVSEvent = true): void {
```

Or something better if you have anything in mind.